apps/openmw/mwphysics/physicssystem.cpp

```diff
@@ -906,6 +906,14 @@ namespace MWPhysics
             return osg::Vec3f();
     }
 
+    osg::Vec3f PhysicsSystem::getOriginalHalfExtents(const MWWorld::ConstPtr &actor) const
+    {
+        if (const Actor* physactor = getActor(actor))
```

Comment:
Use a ternary conditional for compactness.

Reply:
Why though.

Reply:
Why? I don't think it has better readability and it's harder to debug (set breakpoint on specific branch for example). Performance is same for optimized code (look at generated assembly https://godbolt.org/z/iW8hAR).

Reply:
Readability is always better than compactness.